templates/vehicles.html

```diff
@@ -1,43 +1,69 @@
-                </h5>
-                <p>Last location: <bf class="text-capitalize">{{ vehicle.last_location }}</bf>
-                </p>
-                <p>Color: <bf class="text-capitalize">{{ vehicle.color }}</bf>
-                </p>
-                <p>Brand: <bf class="text-capitalize">{{ vehicle.brand }}</bf>
-                </p>
-                <p>Date added: <bf class="text-capitalize">{{ vehicle.date_added }}</bf>
-                </p>
-                {% if vehicle.status == 'available' %}
-                <p class="alert alert-success text-center" role="alert" style="width: 15rem;">Available</p>
-                <form class="form form-horizontal" method="POST" action="/rides/start/{{ vehicle.id }}">
-                    {{ wtf.form_field(form.submit) }}
-                </form>
-                {% else %}
-                <p class="alert alert-warning text-center" role="alert" style="width: 15rem;">Unavailable</p>
-                {% endif %}
-            </div>
-            <div class="col-md" height="500" width="400" style="display: inline-block;">
-                <iframe
-                    src="https://www.google.com/maps/embed/v1/place?q={{ vehicle.last_location }}&key={{ API_KEY }}"></iframe>
+    <div class="row">
+        {% for vehicle in vehicles %}
+        <div class="col-4">
+            <div class="vehicle">
+                <div class="map">{{ vehicle.last_location }}|{{ API_KEY }}|{{vehicle.type}}</div>
+                <div class="content">
+                    <div class="row">
+                        <h5>
+                            <bf class="text-capitalize">{{ vehicle.type }}</bf>
+                        </h5>
+                    </div>
+
+                    <div class="row desc">
+                        <div>
+                            <div class="label">Location</div> 
+                            <bf class="text-capitalize">{{ vehicle.last_location }}</bf>
+                        </div>
+                    </div>
+                    
+                    <div class="row desc">
+                        <div>
+                            <div class="label">Color</div> 
+                            <bf class="text-capitalize">{{ vehicle.color }}</bf>
+                        </div>
+                        <div>
+                            <div class="label">Brand</div> 
+                            <bf class="text-capitalize">{{ vehicle.brand }}</bf>
+                        </div>
+                    </div>
+
+                    <div class="row desc">
+                        <div>
+                            <div class="label">Date added</div> 
+                            <bf class="text-capitalize">{{ vehicle.date_added }}</bf>
+                        </div>
+                    </div>
+
+                    <div class="row start-ride">
+                        <form class="form form-horizontal" method="POST" action="/rides/start/{{ vehicle.id }}">
```

Comment:
> `<form class="form form-horizontal" method="POST" action="/rides/start/{{ vehicle.id }}">`

This should be 

> `<form class="form form-horizontal" method="POST" action="/vehicles/remove/{{ vehicle.id }}">`

since `form_v` is defined in the `user` route (in `server.py`) as `RemoveVehicleForm`.